content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -212,41 +289,161 @@ exhausts the cluster's supply of Pod IPs.
 
 You can find more examples on [Viewing and Setting Quotas](#viewing-and-setting-quotas).
 
-## Quota Scopes
+## Viewing and setting ResourceQuotas
```

Comment:
This section is more suitable for a task page, considering this page is very long already.

Reply:
You are correct, and I agree, but I don't plan to change it in this PR.
> When a PR updates an existing page, you should focus on reviewing the parts of the page that are being updated. That changed content should be reviewed for technical and editorial correctness. If you find errors on the page that don't directly relate to what the PR author is attempting to address, then it should be treated as a separate issue&hellip;

(this is actually moved content; it exists in the current page).

The risk is that I make the PR even larger and it doesn't merge for a long time.

Reply:
I'm fine with postponing this to another PR.